### DISCUSSION TITLE
TI994/A Support

### DISCUSSION THREAD

**Discussion by eharvey71** [webrcade/webrcade#177] [Ideas]
I'm a C64 and TI994/A retro gamer and I'm always searching for better ways to enjoy them on mobile. I absolutely love how you've implemented WebRCade and the newer C64 support. It would be great if there could be something like https://js99er.net/ implemented alongside other computer support.

**Comment by raz0red** [comment]
Yup, agreed. This is one I would definitely like to add in the near future.
